autosar/workspace.py

```diff
@@ -174,16 +174,32 @@ def openXML(self,filename):
         if self.packageParser is None:
             self.packageParser = autosar.parser.package_parser.PackageParser(self.version)
         self._registerDefaultElementParsers(self.packageParser)
+    
+    def openXML(self, filename):
+        xmlroot = parseXMLFile(filename)
+        self.openXML(xmlroot)

```

Comment:
I think this should be
```suggestion
        self._openXML(xmlroot)
```